packages/components/src/components/tooltip/tooltip.js

```diff
@@ -78,6 +78,12 @@ const getMenuOffset = (menuBody, menuDirection) => {
   return undefined;
 };
 
+// @questions
+// -- What is the expected behavior if the user clicks on the trigger when the tooltip is already open
```

Comment:
We used to have keeping it open, or we still have that behavior, but I believe we have a GH issue against it and IIUC we are switching to "toggling" behavior.

Reply:
@asudoh Should I make it toggle in this PR then since I'm doing a lot of updates with the event logic anyways?

Reply:
If so, you don't have to include the change to make it toggle - Separate PR makes more sense. Thank you for asking BTW!

Reply:
Currently in the latest deploy when you click on the trigger when the tooltip is open it flashes close and then opens again.  I put in conditionals testing for the `this.options.classShown` in multiple places but still can not prevent this flash. Do you know how to prevent this?

Reply:
I think we can address that flash thing in a separate PR, given it's _not_ a regression. Here's what happens; Clicking on tooltip trigger causes tooltip body to lose focus, and thus it closes first. And then open-on-click-trigger happens.